src/coord/src/sink_connector.rs

```diff
@@ -377,15 +380,21 @@ async fn build_kafka(
             value_schema,
             ..
         } => {
-            let (key_schema_id, value_schema_id) =
-                publish_kafka_schemas(&ccsr, &topic, &value_schema, key_schema.as_deref())
-                    .await
-                    .context("error publishing kafka schemas for sink")?;
+            let (key_schema_id, value_schema_id) = publish_kafka_schemas(
+                ccsr.as_ref()
+                    .expect("confluent schema registry client must exist"),
+                &topic,
+                &value_schema,
+                key_schema.as_deref(),
+            )
+            .await
+            .context("error publishing kafka schemas for sink")?;
             Some(PublishedSchemaInfo {
                 key_schema_id,
                 value_schema_id,
             })
         }
+        dataflow_types::KafkaSinkFormat::Json => None,
```

Comment:
Did you run into problems with publishing a JSON schema? They're only strings, so I assumed this would be straightforward. I might be too naive, though... :sweat_smile:

Reply:
I did not! Brennan had mentioned that it's okay to skip publishing schemas for the first version, and I wanted to land it sooner (since I'm trying to get everything in before Friday). But I think we should certainly add it next.